pgtransaction/transaction.py

```diff
@@ -108,6 +108,68 @@ def atomic(
     isolation_level=None,
     retry=None,
 ):
+    """
+    Extends django.db.transaction.atomic with PostgreSQL functionality.
+
+    An extension of the django.db.transaction.atomic function
+    which allows one to dynamically set the isolation level when opening a transaction,
+    as well as specifying a retry policy for when an operation in that transaction results
+    in a Postgres locking exception.
+
+    Usage:
+        Since ``pgtransaction.atomic`` inherits from ``django.db.transaction.atomic``, it
+        can be used in exactly the same manner. Additionally, when used as a
+        context manager or a decorator, one can use it to specify the
+        isolation level of the new transaction. For example:
+
+        .. code-block:: python
```

Comment:
For clarity, I would put `import pgtransaction` at the top even if it might be redundant.

Reply:
Maybe just for the first example but not every one